app/src/main/res/layout/content_about_dojma.xml

```diff
@@ -21,5 +21,6 @@
         android:layout_height="match_parent"
         android:layout_below="@+id/about_dojma_image"
         android:layout_marginTop="8dp"
-        android:padding="16dp"/>
+        android:padding="16dp"
+        android:textColor="@color/blue500"/>
```

Comment:
Remove this as the "About DoJMA" will get an update.

Also the colour should be selected so that it is visible on both themes. Because themes are used, adding static colors is discouraged. See how it is done [here](https://github.com/MobileApplicationsClub/DoJMA/blob/master/app/src/main/res/layout/item_format_herald.xml#L18)
 and [here](https://github.com/MobileApplicationsClub/DoJMA/blob/master/app/src/main/res/values/attrs.xml#L16)

The colors for the themes are specified here
[Normal Theme](https://github.com/MobileApplicationsClub/DoJMA/blob/master/app/src/main/res/values/styles.xml#L20)
[Dark Theme](https://github.com/MobileApplicationsClub/DoJMA/blob/master/app/src/main/res/values/styles.xml#L97)